resources/definitions/fdmprinter.def.json

```diff
@@ -3109,28 +3299,59 @@
                             "unit": "mm",
                             "type": "float",
                             "default_value": 1,
-                            "value": "extruderValue(support_interface_extruder_nr, 'support_interface_height')",
+                            "value": "extruderValue(support_bottom_extruder_nr, 'support_interface_height')",
                             "minimum_value": "0",
-                            "minimum_value_warning": "min(3 * resolveOrValue('layer_height'), extruderValue(support_interface_extruder_nr, 'support_bottom_stair_step_height'))",
+                            "minimum_value_warning": "min(3 * resolveOrValue('layer_height'), extruderValue(support_bottom_extruder_nr, 'support_bottom_stair_step_height'))",
                             "maximum_value_warning": "10",
-                            "limit_to_extruder": "support_interface_extruder_nr",
-                            "enabled": "extruderValue(support_interface_extruder_nr, 'support_interface_enable') and support_enable",
+                            "limit_to_extruder": "support_bottom_extruder_nr",
+                            "enabled": "extruderValue(support_bottom_extruder_nr, 'support_bottom_enable') and support_enable",
                             "settable_per_mesh": true
                         }
                     }
                 },
                 "support_interface_skip_height":
                 {
                     "label": "Support Interface Resolution",
-                    "description": "When checking where there's model above the support, take steps of the given height. Lower values will slice slower, while higher values may cause normal support to be printed in some places where there should have been support interface.",
+                    "description": "When checking where there's model above and below the support, take steps of the given height. Lower values will slice slower, while higher values may cause normal support to be printed in some places where there should have been support interface.",
                     "unit": "mm",
                     "type": "float",
                     "default_value": 0.3,
                     "minimum_value": "0",
                     "maximum_value_warning": "support_interface_height",
                     "limit_to_extruder": "support_interface_extruder_nr",
                     "enabled": "extruderValue(support_interface_extruder_nr, 'support_interface_enable') and support_enable",
-                    "settable_per_mesh": true
+                    "settable_per_mesh": true,
+                    "children":
```

Comment:
It doesn't really make sense to split this setting up. That introduces yet 2 more settings, while there are not a lot of models which violate the small gaps heuristic only on overhangs, but not where the support bottoms would be.

Please just use one setting for this.